homeassistant/components/filesize/sensor.py

```diff
@@ -39,11 +43,23 @@ def setup_platform(
     setup_reload_service(hass, DOMAIN, PLATFORMS)
 
     sensors = []
+    pathlist = []
```

Comment:
```suggestion
    pathlist = set()
```

Reply:
It probably doesn't matter in this case since the user is unlikely to have 1000s of path sensors, but you never know...

We don't want a linear search of a list each loop.

Reply:
Sorry I didn't see it before